includes/admin/class-admin.php

```diff
@@ -164,8 +167,12 @@ public function display_settings_page() {
 			<h2><?php echo esc_html( $this->page_title ); ?></h2>
 			<p>
 				<?php
-				/* translators: TOC is table of contents */
-				esc_html_e( 'Specify default settings for HM Content TOC plugin', 'hm-content-toc' );
+				/* translators: TOC is table of contents. 1: Opening documentation link tag 2: Closing documentation link tag */
+				printf(
+					esc_html__( 'Specify default settings for HM Content TOC plugin. For more information and usage %1$ssee documentation on github website%2$s.', 'hm-content-toc' ),
+					'<a href="' . esc_url( self::GITHUB_DOC_URL ) . '" target="_blank">',
+					'</a>'
```

Comment:
@johnbillion not sure if placeholders are fine here? Is there a better solution for this case, where I want to make part of the sentence as a link?